### PR TITLE
Security issue: Ship with a default implementation for _getattr_

### PR DESCRIPTION
**Pull request by icemac**
It prevents from using the ``format()`` method on str/unicode as it is not
safe, see: http://lucumr.pocoo.org/2016/12/29/careful-with-str-format/

This fix supersedes #80.

### CODE REVIEW

**Comment by stephan-hof** [comment]
This default implementation makes sense to me.

File: src/RestrictedPython/Guards.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # AccessControl.ZopeGuards contains a large set of wrappers for builtins.
 # DocumentTemplate.DT_UTil contains a few.
 
+from . import _compat
```
Comment (loechel) [inline src/RestrictedPython/Guards.py]: sorry but I do not like local imports, please use absolute imports.

**Comment by icemac** [comment]
@loechel The code used by Zope to allow `format()` is in zopefoundation/AccessControl#46.

File: src/RestrictedPython/Guards.py (hunk 1 above)
Comment (icemac) [reply src/RestrictedPython/Guards.py]: Fixed.

**Pull request merged**